mesa/visualization/ModularVisualization.md

```diff
@@ -1,32 +1,31 @@
+Modular visualization is one of Mesa's core features. Mesa is designed to provide predefined visualization modules, which can be easily subclassed for your needs, and mixed-and-matched to visualize your particular model. (Some day, Mesa hopes to host a wide variety.) This document describes how to use and create new visualization modules.
 
 ## Overview
 
-An interactive modular visualization is a set of **Elements**, each of which is an instance of a visualization **Module**. To visualize a model, create a new ModularServer with a list of the elements you want to visualize, in the order you want them to appear on the visualization web page. 
+An interactive modular visualization is a set of **Elements**, each of which is an instance of a visualization **Module**. To visualize a model, create a new ModularServer with a list of the elements you want to visualize, in the order you want them to appear on the visualization web page.
 
-For example, if you have a model MyModel, and two elements, *canvas_vis* and *graph_vis*, you would create a visualization with them via:
-```python    
+For example, if you have a model `MyModel`, and two elements, *canvas_vis* and *graph_vis*, you would create a visualization with them via:
+```python
     server = ModularServer(MyModel, [canvas_vis, graph_vis])
     server.launch()
 ```
-And then open your browser to http://127.0.0.1:8888/ in order to open it. If you prefer a different port, for example 8887, you can pass it to the server:
+Then you will be able to view the elements in your browser at http://127.0.0.1:8888/. If you prefer a different port, for example 8887, you can pass it to the server as an argument.
 ```python
     server.launch(8887)
 ```
-Under the hood, each visualization module consists of two parts: Python code which can take a model object and renders it into some data describing the visualization, and a JavaScript object which in turn receives data from the Python render (via the ModularServer) and actually draws it in the browser. 
-
-Phew.
+Under the hood, each visualization module consists of two parts:
 
-Okay, so what do you actually need to know?
+1. **Data rending** - Python code which can take a model object and renders it into some data describing the visualization.
+2. **Browser rendering** - JavaScript object which in turn receives data from the Python render (via the ModularServer) and actually draws it in the browser.
 
 ## Using Pre-Built Modules
 
-Most of the time, you should be able to build your visualization using pre-built modules. To do this, you don't need to worry about HTML and JavaScript at all, and only look at the documentation for the particular module you want to use.
+Part of the Mesa library are pre-built modules. Using the built-ins allow you to build a visualization without worrying about the HTML and javascript. Consult the documention for a variety of modules.
```

Comment:
First sentence on Line 26 can be replaced with this.
"Mesa already comes with some pre-built modules."